package.json

```diff
@@ -20,7 +20,7 @@
     "test": "jest",
     "format": "eslint --ignore-path=.gitignore --ext .ts,.tsx --fix . && prettier --ignore-path=.gitignore --loglevel=error --write .",
     "check-eslint": "eslint --ignore-path=.gitignore --ext .ts,.tsx --max-warnings=0 .",
-    "check-prettier": "prettier --ignore-path=.gitignore --list-different .",
+    "check-prettier": "prettier --ignore-path=.gitignore --list-different --write .",
```

Comment:
Let's not change it please, this command only checks. If you need prettier to write to your files, you can use `format` command.